peps/converters.py

```diff
@@ -140,6 +150,21 @@ def convert_pep_page(pep_number, content):
     # Fix PEP links
     pep_content = BeautifulSoup(data['content'], 'lxml')
     body_links = pep_content.find_all("a")
+    # Fix highlighting code
+    code_blocks = pep_content.find_all('pre', class_='code')
+    for cb in code_blocks:
+        del cb['class']
+        div = pep_content.new_tag('div')
+        div['class'] = ['highlight']
+        cb.wrap(div)
+    # Highlight existing pure-Python PEPs
```

Comment:
This chunk is dealing with blocks that don't have an explicit language, those marked with `::`, and defaulting to Python.

This PR is being cautious and only defaulting to Python for `PURE_PYTHON_PEPS`.

More detail:

It finds all `<pre class="literal-block">` (the language-specific ones have just had their `literal-block` removed), then puts it through the Python highlighter.

---

I think it'd be good to default to Python for all `::` blocks, and we can specify the language in the actual PEPs for non-Python `::` blocks.

Sphinx also defaults to Python*, so this will also help for when Sphinx/RtD stuff is ready, and avoid us needing to replace `::` with `.. code-block:: python`.

\* [Sphinx defaults to Python](https://www.sphinx-doc.org/en/1.5/markup/code.html) unless a directive like `highlight:: c`  is used. The PEPs don't use `highlight::` so we're good.